test/timezones/accessors.jl

```diff
@@ -0,0 +1,29 @@
+import Base.Dates: Second
+
+warsaw = resolve("Europe/Warsaw", tzdata["europe"]...)
+fixed = FixedTimeZone("Fixed", -7200, 3600)
+
+# Determine total offset for FixedTimeZone.
+@test TimeZones.offset(fixed) == Second(-3600)
+@test_throws MethodError TimeZones.offset(warsaw)
+
+# ZonedDateTime accessors
+zdt = ZonedDateTime(DateTime(2014,6,12,23,59,58,57), fixed)
+@test TimeZones.localtime(zdt) == DateTime(2014,6,12,23,59,58,57)
+@test TimeZones.utc(zdt) == DateTime(2014,6,13,0,59,58,57)
+
+@test TimeZones.days(zdt) == 735396
+@test TimeZones.hour(zdt) == 23
```

Comment:
`ZonedDateTime` by default expects you are passing in a `DateTime` that is in the given `TimeZone`. Internally the `ZonedDateTime` converts that localtime into utc. Note the difference in the tests that convert the `zdt` into `utc` or `localtime`.